sda-commons-server-mongo-testing/src/test/java/org/sdase/commons/server/mongo/testing/UseExistingMongoDbRuleTest.java

```diff
@@ -5,16 +5,17 @@
 import com.mongodb.MongoClient;
 import com.mongodb.client.FindIterable;
 import org.bson.Document;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.BeforeClass;
 import org.junit.ClassRule;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 import org.junit.runner.Description;
 import org.junit.runners.model.Statement;
 import org.sdase.commons.server.testing.SystemPropertyRule;
 
-public class UseExistingMongoDbRuleTest {
+class UseExistingMongoDbRuleTest {
 
   @ClassRule
   public static final MongoDbRule EXTERNAL_DB =
```

Comment:
Keeping the old JUnit 4 annotation looks odd. I'd prefer reverting the test to the old JUnit 4 state. We will remove it when we remove all our provided JUnit 4 rules.